graphics/surface.h

```diff
@@ -134,223 +145,234 @@ struct Surface {
-	 * @param penX The width of the pen (thickness in the x direction)
-	 * @param penY The height of the pen (thickness in the y direction)
-	 * @param color The color of the line.
-	 * @note This is just a wrapper around Graphics::drawThickLine
-	 * @note The x/y coordinates of the start and end points are the upper-left most part of the pen
+	 * @param x0     The x coordinate of the start point.
+	 * @param y0     The y coordinate of the start point.
+	 * @param x1     The x coordinate of the end point.
+	 * @param y1     The y coordinate of the end point.
+	 * @param penX   Width of the pen (thickness in the x direction).
+	 * @param penY   Height of the pen (thickness in the y direction).
+	 * @param color  Color of the line.
+	 *
+	 * @note This is just a wrapper around Graphics::drawThickLine.
+	 *
+	 * @note The x/y coordinates of the start and end points are the upper leftmost part of the pen.
 	 */
 	void drawThickLine(int x0, int y0, int x1, int y1, int penX, int penY, uint32 color);
 
 	/**
 	 * Draw a horizontal line.
 	 *
-	 * @param x The start x coordinate of the line.
-	 * @param y The y coordiante of the line.
-	 * @param x2 The end x coordinate of the line.
-	 *           In case x > x2 the coordinates are swapped.
-	 * @param color The color of the line.
+	 * @param x      The start x coordinate of the line.
+	 * @param y      The y coordinate of the line.
+	 * @param x2     The end x coordinate of the line.
+	 *               In case x > x2, the coordinates are swapped.
+	 * @param color  Color of the line.
 	 */
 	void hLine(int x, int y, int x2, uint32 color);
 
 	/**
 	 * Draw a vertical line.
 	 *
-	 * @param x The x coordinate of the line.
-	 * @param y The start y coordiante of the line.
-	 * @param y2 The end y coordinate of the line.
-	 *           In case y > y2 the coordinates are swapped.
-	 * @param color The color of the line.
+	 * @param x      The x coordinate of the line.
+	 * @param y      The start y coordinate of the line.
+	 * @param y2     The end y coordinate of the line.
+	 *               In case y > y2, the coordinates are swapped.
+	 * @param color  Color of the line.
 	 */
 	void vLine(int x, int y, int y2, uint32 color);
 
 	/**
 	 * Fill a rect with a given color.
 	 *
-	 * @param r Rect to fill
-	 * @param color The color of the rect's contents.
+	 * @param r      The rectangle to fill.
+	 * @param color  The color to fill the rect with.
 	 */
 	void fillRect(Common::Rect r, uint32 color);
 
 	/**
 	 * Draw a frame around a specified rect.
 	 *
-	 * @param r Rect to frame
-	 * @param color The color of the frame.
+	 * @param r      The rectangle to frame.
+	 * @param color  The color of the frame.
 	 */
 	void frameRect(const Common::Rect &r, uint32 color);
 
-	// See comment in graphics/surface.cpp about it
+	/**
+	 * Move the surface horizontally or vertically.
+	 *
+	 * See @c surface.cpp for the implementation.
+	 * @todo More explanation required?
```

Comment:
It would be more accurate to say that it moves the surface content horizontally and/or vertically by the given number of pixels. And I don't think you need more explanation than that.